src/components/Global/Header.vue

```diff
@@ -131,6 +131,12 @@ export default {
       return Boolean(this.env && this.apiUrl)
     },
   },
+  mounted() {
+    document.body.style.paddingTop = 'var(--topbar-height)'
+  },
+  destroyed() {
+    document.body.style.paddingTop = ''
+  },
```

Comment:
Instead setting paddingTop to `''` on destroy, I would set the initial value (one before `mounted` occurred), as it may be hard to debug in the future, if something else would assign this value
```suggestion
  mounted() {
    this.initialBodyPaddingTop = document.body.style.paddingTop
    document.body.style.paddingTop = 'var(--topbar-height)'
  },
  destroyed() {
    document.body.style.paddingTop = this.initialBodyPaddingTop
  },
```